pandas/io/tests/test_common.py

```diff
@@ -90,6 +90,15 @@ def test_iterator(self):
         expected.index = [0 for i in range(len(expected))]
         tm.assert_frame_equal(concat(it), expected.iloc[1:])
 
+    def test_temporary_file(self):
+        from tempfile import TemporaryFile
+        new_file = TemporaryFile("w+")
+        new_file.write("0   0")
+        new_file.flush()
+        new_file.seek(0)
+
+        dataframe = read_table(new_file, sep=r"\s+", header=None, engine="python")
+
 
```

Comment:
Some of the errors I get:

nosetests -s pandas\io\tests\parser\python_parser_only.py:PythonParserTests
EEEEEEEE
======================================================================
ERROR: pandas.io.tests.parser.python_parser_only.PythonParserTests.test_BytesIO_input
----------------------------------------------------------------------
Traceback (most recent call last):
  File "G:\Anaconda3\lib\site-packages\nose\case.py", line 198, in runTest
    self.test(*self.arg)
  File "G:\Informatique\pandas\pandas\io\tests\parser\python_parser_only.py", line 84, in test_BytesIO_input
    result = self.read_table(data, sep="::", encoding='cp1255')
AttributeError: 'PythonParserTests' object has no attribute 'read_table'

======================================================================
ERROR: pandas.io.tests.parser.python_parser_only.PythonParserTests.test_decompression_regex_sep
----------------------------------------------------------------------
Traceback (most recent call last):
  File "G:\Anaconda3\lib\site-packages\nose\case.py", line 198, in runTest
    self.test(*self.arg)
  File "G:\Informatique\pandas\pandas\io\tests\parser\python_parser_only.py", line 133, in test_decompression_regex_sep
    data = open(self.csv1, 'rb').read()
AttributeError: 'PythonParserTests' object has no attribute 'csv1'

======================================================================
ERROR: pandas.io.tests.parser.python_parser_only.PythonParserTests.test_negative_skipfooter_raises
----------------------------------------------------------------------
Traceback (most recent call last):
  File "G:\Anaconda3\lib\site-packages\nose\case.py", line 198, in runTest
    self.test(*self.arg)
  File "G:\Informatique\pandas\pandas\io\tests\parser\python_parser_only.py", line 34, in test_negative_skipfooter_raises
    self.read_csv(StringIO(text), skipfooter=-1)
AttributeError: 'PythonParserTests' object has no attribute 'read_csv'

And they are proper issues in the test file.